configs/configgen.sh

```diff
@@ -3,14 +3,19 @@
 set -e
 
 SCRIPT_DIR=`dirname $0`
-BUILD_DIR=$2/configgen
+BUILD_DIR=/tmp/configgen
+OUT_DIR=$1
```

Comment:
Let me try this again, but I believe I had problem executing the venv out of OUT_DIR in sandbox mode which is why I did /tmp in the first place. If that's the case, should I use /tmp? Or do a local test?